include/oneapi/tbb/concurrent_queue.h

```diff
@@ -215,6 +251,16 @@ class concurrent_queue {
 
     queue_allocator_type my_allocator;
     queue_representation_type* my_queue_representation;
+
+    friend bool operator==( const concurrent_queue<T, Allocator>& lhs,
+                            const concurrent_queue<T, Allocator>& rhs ){
+      return lhs.my_queue_representation == rhs.my_queue_representation;
+    }
+  
+    friend bool operator!=( const concurrent_queue<T, Allocator>& lhs,
+                            const concurrent_queue<T, Allocator>& rhs ){
+      return lhs.my_queue_representation != rhs.my_queue_representation;
+    }
```

Comment:
```suggestion
#if !__TBB_CPP20_COMPARISONS_PRESENT
    friend bool operator!=( const concurrent_queue<T, Allocator>& lhs,
                            const concurrent_queue<T, Allocator>& rhs ){
      return lhs.my_queue_representation != rhs.my_queue_representation;
    }
#endif // __TBB_CPP20_COMPARISONS_PRESENT
```

Since C++20, `!=` operation would be automatically done by the compiler using the `==` primary operator